.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
```

Comment:
Did we set up travis for this project?

Reply:
not yet, kept it just in future if we need to

Reply:
We can remove this for now. We can add whenever we wish to add Travis.